cloud/pod_util.go

```diff
@@ -335,3 +356,102 @@ func exportPodEnvVars(smConf evergreen.SecretsManagerConfig, opts pod.TaskContai
 
 	return allEnvVars
 }
+
+// GetFilteredResourceIDs gets at most n resources that match the given resource
```

Comment:
Can we refer to n as limit since that is how it's described in the function definition?